packages/main/src/WheelSlider.hbs

```diff
@@ -19,14 +19,25 @@
 	<div id="{{this._id}}--selection-frame" class="ui5-wheelslider-selection-frame"></div>
 		<div id="{{this._id}}--wrapper" class="ui5-wheelslider-wrapper">
 			{{#if expanded}}
-				<ul id="{{this._id}}--items-list">
+				<ul id="{{this._id}}--items-list" role="listbox" aria-labelledby="{{label}}">
```

Comment:
Still aria-labelledby="{{label}}" is used here line 22 WheelSlider.hbs

Reply:
There was an if statement with a variation of the ul element. Aria-label is changed in both places now.